Source/WebCore/platform/ScrollbarsController.h

```diff
@@ -109,6 +110,8 @@ class ScrollbarsController {
 
     WEBCORE_EXPORT virtual void updateScrollbarStyle() { }
 
+    WEBCORE_EXPORT virtual bool isRemoteScrollbarsController() { return false; }
```

Comment:
You should add `SPECIALIZE_TYPE_TRAITS` support on ScrollbarsController, so that the caller can just use `is<RemoteScrollbarsController>()`

Reply:
Fixed.